src/components/common/FileUpload.js

```diff
@@ -1,40 +1,41 @@
 import React, { useState } from 'react';
-import axios from 'axios';
-import { Button } from 'antd';
+// import axios from 'axios';
+import { Image } from 'antd';
 import './FileUpload.css';
-import { useOktaAuth } from '@okta/okta-react';
+// import { useOktaAuth } from '@okta/okta-react';
 
 const FileUpload = ({ uploadUrl }) => {
-  const { authState } = useOktaAuth();
+  // const { authState } = useOktaAuth();
   const [selectedFile, setSelectedFile] = useState();
 
-  const getAuthHeader = authState => {
-    if (!authState.isAuthenticated) {
-      throw new Error('Not authenticated');
-    }
-    return { Authorization: `Bearer ${authState.idToken}` };
-  };
+  // const getAuthHeader = authState => {
```

Comment:
same with all the code in this file, if it needs to be commented out, leave a comment next to them as to why, if they do not need to be there then just delete them.